### PR TITLE
docs for aggregation in UCR

### PR DESCRIPTION
**Pull request by czue**
@NoahCarnahan, no test coverage

### CODE REVIEW

File: corehq/apps/userreports/README.md [hunk 1]
```diff
@@ -616,6 +616,23 @@ numeric_percent | Percentage as a number                         | 33
...
+### The "aggregation" column property
+
+The aggregation column property defines how the column should be aggregated.
+If the report is not doing any aggregation, or if the column is one of the aggregation columns this should always be `"simple"` (see [Aggregation](#aggregation) below for more information on aggregation).
+
+The following table documents the other aggregation options, which can be used in aggregate reports.
+
+Format          | Description
+--------------- | -----------------------------------------------
+simple          | No aggregation
```
Comment (NoahCarnahan) [inline corehq/apps/userreports/README.md]: I can never seem to remember: Do columns with `"aggregation": "simple"` get an arbitrary value from the grouped rows, or is that an error?

Comment (czue) [inline corehq/apps/userreports/README.md]: I think it's an error

Comment (czue) [inline corehq/apps/userreports/README.md]: (also not sure off the top of my head though)

**Comment by czue** [comment]
@NoahCarnahan any reason you're not merging?

**Comment by NoahCarnahan** [comment]
No reason. I meant to follow up on the "simple" aggregation thing, but it fell off my radar.

**Pull request merged**